src/components/Roadmap/data/iacmData.ts

```diff
@@ -52,7 +52,7 @@ export const IacmData: Horizon = {
     ],
   },
   Now: {
-    description: "Q4 2024, Nov 2024-Jan 2025",
+    description: "Q1 2025, Feb 2025-April 2025",
```

Comment:
```suggestion
    description: "Q1 2025, Feb 2025-Apr 2025",
```

Just to following the same convention as the other dates